src/ImageSharp/Image/PixelArea{TColor}.cs

```diff
@@ -29,6 +29,11 @@ public sealed unsafe class PixelArea<TColor> : IDisposable
         private IntPtr dataPointer;
 
         /// <summary>
+        /// True if <see cref="Bytes"/> was rented from <see cref="BytesPool"/> by the constructor
+        /// </summary>
+        private bool isBufferRented;
```

Comment:
Yeah, but I still don't like the idea of controlling a major aspect of a class with a bool parameter. It was a lazy solution from my side, I'd like to see something like `PixelArea<T>.CreateWithRentedBuffer()` instead. What do you think?